core/src/main/scala/net/ruippeixotog/scalascraper/browser/JsoupBrowser.scala

```diff
@@ -48,6 +48,14 @@ class JsoupBrowser(val userAgent: String = "jsoup/1.8") extends Browser {
     using(inputStream) { _ => JsoupDocument(Jsoup.parse(inputStream, charset, "")) }
 
   def cookies(url: String) = cookieMap.toMap
+ 
+  def setCookie(url: String, key: String, val: String) = {
+    cookieMap ++= List(key -> val)
+  }
+ 
+  def setCookies(url: String, m: Map[String, String]) = {
+    cookieMap ++= m.toList
```

Comment:
thanks; guess I somehow thought the mutable/immutable difference might matter. good to know!